tests/components/rflink/test_light.py

```diff
@@ -281,19 +280,21 @@ async def test_signal_repetitions_cancelling(hass: HomeAssistant, monkeypatch) -
 
     # setup mocking rflink module
     _, _, protocol, _ = await mock_rflink(hass, config, DOMAIN, monkeypatch)
+    await hass.async_block_till_done()
```

Comment:
This test was testing event loop timing. It should likely be replaced in the future